File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/DownloadQsBackup200Response.scala

package org.openapitools.client.model


case class DownloadQsBackup200Response (
    _text: Option[String],
    /* A pre-signed download URL valid for 24 hours. */
    _url: Option[String]
)
object DownloadQsBackup200Response {
    def toStringBody(var_text: Object, var_url: Object) =
        s"""
        | {
        | "text":$var_text,"url":$var_url
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/DownloadQsBackup200Response.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class DownloadQsBackup200Response
{
    #[DTA\Data(field: "text", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $text = null;

    /**
     * A pre-signed download URL valid for 24 hours.
     */
    #[DTA\Data(field: "url", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $url = null;

}
